jablib/src/main/java/org/jabref/model/entry/types/BiblatexEntryTypeDefinitions.java

```diff
@@ -16,6 +16,17 @@
  */
 public class BiblatexEntryTypeDefinitions {
 
+    static final BibEntryType MISC = new BibEntryTypeBuilder()
```

Comment:
Please move to the old position after article, makes it easer to review the changes and I don't think there are some